app/components/Image.js

```diff
@@ -22,7 +22,10 @@ export default class Image extends Component {
     onMount: PropTypes.func,
     onLoad:  PropTypes.func,
     onError: PropTypes.func,
-    style:   PropTypes.array
+    style:   PropTypes.oneOfType([
+      PropTypes.array,
```

Comment:
strangely, I think so, ha. that was already set. 

I added object as a precautionary measure in case the component was indeed receiving an array from somewhere else.